runtime/storage.go

```diff
@@ -176,23 +162,6 @@ func (s *Storage) GetDomainStorageMap(
 		}
 	}()
 
-	if !s.Config.StorageFormatV2Enabled {
-
-		// When StorageFormatV2 is disabled, handle all accounts as v1 accounts.
-
-		// Only read requested domain register.
-
-		domainStorageMap = s.getDomainStorageMapForV1Account(
-			address,
-			domain,
-			createIfNotExists,
-		)
-
-		return
-	}
-
-	// StorageFormatV2 is enabled.
-
 	// Check if cached account format is available.
```

Comment:
Unfortunately we won't be able to remove the read of the domain register / V1 reads here, because that will produce a different execution result, i.e. require a HCU.